src/Element/Service/ElementSaveService.php

```diff
@@ -40,8 +39,7 @@ public function __construct(
     }
 
     /**
-     * @throws DuplicateFullPathException
-     * @throws Exception
+     * @inheritdoc
```

Comment:
```suggestion
     * {@inheritdoc}
```
TBH i have no idea what's the difference, but docs says it should be with {}
https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

Reply:
Hmm I did not see, that, at some places, e.g here we also have it like `inheritDoc`
https://github.com/pimcore/studio-backend-bundle/blob/1.x/src/Element/Service/ElementLocationService.php#L52

I will replace them to the format you posted

Reply:
At least we should keep it consistent...

Reply:
now they should be consistent :)